moped-toolbox/pg_replicate_to_local/scripts/pull_down_remote_db.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo "Downloading remote DB"
+pg_dump --verbose --no-owner > /root/workspace/db.sql
```

Comment:
I got this error:
```
➜  pg_replicate_to_local git:(flh_database_replication_tool) scripts/pull_down_remote_db.sh
Downloading remote DB
scripts/pull_down_remote_db.sh: line 4: /root/workspace/db.sql: No such file or directory
```

Reply:
Thanks! I had a typo in my instructions in the PR, thank you for your help!